api/models/Comment.js

```diff
@@ -60,44 +55,34 @@ module.exports = bookshelf.Model.extend(Object.assign({
     return Promise.resolve([])
   },
 
-  createActivities: function (trx) {
-    const isReplyToPost = !this.get('comment_id')
-
-    var toLoad = ['post', 'post.followers']
-    if (!isReplyToPost) toLoad = toLoad.concat(['comment', 'comment.followers'])
-
```

Comment:
What I took out was the logic related to nested comments -- it was never used in production so I figured it would be cleaner to throw it out because if and when we want to use it again, it'll be done differently anyway with GroupConnection